platformio.ini

```
@@ -133,6 +133,14 @@ board           = esp-wrover-kit
 monitor_speed   = 115200
 upload_speed    = 1500000
 
+[dev_lilygo-tdisplay-s3]
+extends         = dev_esp32-s3
+build_flags     = -DLILYGOTDISPLAYS3=1
+                  -DPIN_SDA=16
+                  -DTOGGLE_BUTTON_1=0
+                  -DTOGGLE_BUTTON_2=14
+                  ${base.build_flags}
```

Please build on top of the build flags of the section (device) you're extending:
 
```suggestion
                  ${dev_esp32-s3.build_flags}
```

That way, build flags can be added to the base section/device without then having to consider sections extending from it. (The fact dev_esp32-s3 currently sets no `build-flags` of its own is irrelevant; it will automatically inherit those from `base`).

---

Good catch; I agree. Done.

I also changed the default data pin on this device because once WiFi is enabled the previous pin becomes unstable and results in sporadic led flashing.